permissions.mm

```diff
@@ -367,11 +367,23 @@ void AskForFullDiskAccess(const Napi::CallbackInfo &info) {
 
 // Request Screen Capture Access.
 void AskForScreenCaptureAccess(const Napi::CallbackInfo &info) {
-  if (@available(macOS 10.14, *)) {
-    NSWorkspace *workspace = [[NSWorkspace alloc] init];
-    NSString *pref_string = @"x-apple.systempreferences:com.apple.preference."
-                            @"security?Privacy_ScreenCapture";
-    [workspace openURL:[NSURL URLWithString:pref_string]];
+  if (@available(macOS 10.15, *)) {
```

Comment:
Just making sure but Catalina (10.15) is the first instance of screen recording permissions and that's why you bumped this right?

Reply:
It is.